src/nad2bin.c

```diff
@@ -3,6 +3,7 @@
 #include <stdlib.h>
 
 #define PJ_LIB__
+#include <proj_internal.h>
```

Comment:
`proj_internal.h` is private to PROJ.4 so should be included as such with `#include "proj_internal.h"`. `projects.h` is going to be removed from the public API in the near future, so we are changing the include style of that too whenever we come across it.

Reply:
No problem. What about the order of the includes? That is a bit inconsistent. In `nad2bin.c` it would then be:

```cpp
#include <stdio.h>
#include <stdlib.h>

#define PJ_LIB__
#include "proj_internal.h"
#include "projects.h"
```

In `pj_gridinfo.c`:

```cpp
#define PJ_LIB__

#include "proj_internal.h"
#include "projects.h"
#include <string.h>
#include <math.h>
#include <errno.h>
```

Reply:
I tend to put the `""`-includes at the bottom of the list, but as you've discovered there's not much consistency in the existing code.